Ambit/Source/Ambit/Utils/AmbitWorldHelpers.spec.cpp

```diff
@@ -997,6 +997,39 @@ void AmbitWorldHelpersSpec::Define()
             TestComponentActor->SetActorLocation(FVector(0, 0, 0));
             TestEqual("The array is empty", Transforms.Num(), 0);
         });
+
+        It("Will generate objects relative to spline rotation if bFollowSplineRotation is true", [this]()
+        {
+            const bool bFollowSplineRotation = true;
+
+            const float RotationMin = 120.0;
+            const float RotationMax = 120.0f;
+            RealSpline->SetLocationAtSplinePoint(1, FVector(0, 1000, 0), ESplineCoordinateSpace::World);
+
+            const TArray<AActor*> ActorsToSearch;
+            const int32 RandomSeed = 0;
+            const float DensityMin = 1.0f;
+            const float DensityMax = 1.0f;
+
+            const TArray<FTransform>& Transforms = AmbitWorldHelpers::GenerateRandomLocationsFromSpline(
+                RealSpline, ActorsToSearch, RandomSeed, false, DensityMin,
+                DensityMax, RotationMin, RotationMax, bFollowSplineRotation);
+            UE_LOG(LogAmbit, Display, TEXT("The number of Transforms is: %d"), Transforms.Num());
+
+            const float SplinePointRotation = RealSpline->GetRotationAtSplinePoint(1, ESplineCoordinateSpace::World).Yaw;
+            const float ExpectedRotation = RotationMin + SplinePointRotation;
```

Comment:
I would recommend to hard-code Expected values, rather than rely on computation or other functions to generate it. This can remove the reliance of 1019.

Reply:
Got it.